gas-oracle/tokenprice/tokenprice.go

```diff
@@ -207,7 +208,7 @@ func determineTokenPairForMNT(tokenPairMNTMode bool) string {
 	}
 }
 
-func getMedian(nums ...float64) float64 {
+func getMedian(nums []float64) float64 {
 	sort.Float64s(nums)
 	return nums[len(nums)/2]
 }
```

Comment:
Would directly iterating through the list to find the median be more efficient than sorting the list first and then retrieving the element at the middle position?